pkg/models/result.go

```diff
@@ -68,7 +68,7 @@ func (nr *NodeResults) GetErrorHostMap() map[string]error {
 	return hostErrors
 }
 
-func (nr *NodeResults) HasNodeIDWithError(nodeID string) bool {
+func (nr *NodeResults) HasIDWithError(nodeID string) bool {
```

Comment:
nit: I can see this func used for NodeIDs and NodeIPs, I feel like it should be `func  HasError(id string)` 
as id would cover NodeID and Node IP